### ISSUE TITLE
ScreenToWorld Gives Wrong Value

### ISSUE THREAD

**Issue by coman3** [EloBuddy/EloBuddy.Issues#9]
ScreenToWorld is (from what i can see) not taking zoom into account. In the below image, the center of the crosshair is the screen position, and the end of the red line, is the crosshairs "ScreenToWorld" value. (Note: the start of the red line is Player.Instance.Position)
![image](https://cloud.githubusercontent.com/assets/1707575/12534631/604319aa-c2b7-11e5-9a03-02d1ce9526ca.png)

the code for the drawing of above is as so:

``` javascript
Crosshair = new Vector2[5];
            Crosshair[0] = new Vector2(Drawing.Width / 2f - CrosshairSize, Drawing.Height / 2f);
            Crosshair[1] = new Vector2(Drawing.Width / 2f + CrosshairSize, Drawing.Height / 2f);
            Crosshair[2] = new Vector2(Drawing.Width / 2f, Drawing.Height / 2f);
            Crosshair[3] = new Vector2(Drawing.Width / 2f, Drawing.Height / 2f + CrosshairSize);
            Crosshair[4] = new Vector2(Drawing.Width / 2f, Drawing.Height / 2f - CrosshairSize);
Line.DrawLine(Color.Black, Crosshair);
Line.DrawLine(Color.Red, Player.Instance.Position, Crosshair[2].ScreenToWorld());
```

when you zoom IN as far as you can, the position is much closer:
![image](https://cloud.githubusercontent.com/assets/1707575/12534630/5090e9ba-c2b7-11e5-84b8-3b584c046b24.png)

**Comment by Hellsing** [comment]
This is a known issue for quite some time now, I've assigned @finndev to this issue as it's a core bug. Thanks for reporting this.

**Comment by finndev** [comment]
fixed in the patch 6.3 binaries, thank you for reporting
